conda_build/build.py

```diff
@@ -432,6 +432,10 @@ def write_info_json(m):
             info_index['depends'] = [' '.join(dist.split('::', 1)[-1].rsplit('-', 2))
                                      for dist in dists]
 
+    # change index.json timestamp for reproducibility
+    if 'timestamp' in info_index:
+        info_index['timestamp'] = 86400
```

Comment:
Yep, what I ended up doing was adding 1 to the DUMMY_TIMESTAMP for the pyc files.